zeton/api/points.py

```diff
@@ -10,23 +10,23 @@
 from zeton.data_access import users
 
 
-def is_daily_limit_reached(child_id, exercise_id):
-    now = datetime.datetime.now() - datetime.timedelta(days=1)
+def is_limit_reached(child_id, exercise_id, days, max_column):
+    now = datetime.datetime.now() - datetime.timedelta(days= days)
     dt_string = datetime.datetime.fromisoformat(str(now))
     history = zeton.data_access.points.get_points_history_limits(child_id, dt_string, exercise_id)
     points_events_count = history.__len__()
 
     if points_events_count > 0:
-        day_limit = history[0]['max_day']
-        if points_events_count >= day_limit:
+        limit = history[0][max_column]
```

Comment:
Po przekroczeniu limitu tygodniowego wywala błąd. `IndexError: No item with that key`
Nie ma klucza `max_week` w tej odpowiedzi z bazy danych.

Reply:
Leszek możesz nagrać mi tu PSR (Problem step recorder, od momentu recreate bazy danych)? Próbowałem odtworzyć problem i coś mi nie idzie.

Reply:
Problem występował w momencie gdy osiągneliśmy limit tygodniowy czyli `max_week`, natomiast nie został osiągniety limit dzienny `max_day`. Można go w prosty sposób wywołać, gdy zadeklarujesz zadanie, gdzie limit  `max_day` jest większy od limitu `max_week`. W ostnim commicie zostało to naprawione.